index.html

```diff
@@ -0,0 +1,45 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dienebi Ombu - Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Poppins'>
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <a class="logo" href="index.html">ebidien</a>
+            <div class="main-menu">
+                <img src="menu-button.svg" alt="Main Menu">
+            </div>
+        </nav>
+    </header>
+    <section class="headline">
+        <div>
+            <div class="intro-title">
+                <h1>I'm Dienebi. Glad to see you!</h1>
+            </div>
+            <div class="intro-body">
+                <p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
+                <h2>LET'S CONNECT</h2>
+            </div>
+            <ul class="social-media-icons">
+                <li class="social-media-icon"><a href="https://twitter.com/ebidien83"><img src="twitter.png"
+                            alt="twitter" /></a></li>
+                <li class="social-media-icon"><a href="https://www.linkedin.com/in/ebidien"><img src="linkedin.png"
+                            alt="linkedin" /></a></li>
+                <li class="social-media-icon"><a href="https://medium.com/@ebidien"><img src="medium.png"
+                            alt="medium" /></a></li>
+                <li class="social-media-icon"><a href="https://github.com/ebidien"><img src="github.png"
+                            alt="github" /></a></li>
+                <li class="social-media-icon"><a href="https://angel.co/u/dienebi-ombu"><img src="angellist.png"
+                            alt="angellist" /></a></li>
+            </ul>
+        </div>
+    </section>
+</body>
```

Comment:
👏 Your design is awesome, except that we need to work on a few missing details 📝 .
- [ ] Let's consider some current differences between your design and the figma design:
- Your design:
![image](https://user-images.githubusercontent.com/99715640/211960036-dda11e84-0ed1-49af-9567-07bfc0a352a8.png)

- Figma design:
![image](https://user-images.githubusercontent.com/99715640/211960091-d9abfc95-f400-405a-8a2e-87e4953591a0.png)

- [ ] Somehow, it looks like your `h1` style properties are not the same with the requirement. Kindly check the `font-size`, `font-weight` , etc.
- [ ] There are some slight differences in the `background-images` of both designs. Let's make yours look even more closely like the design so that the beauty of your design-sense can really shine outstandingly. Did you see how colorful the template is, imagine yours looks exactly that way too?
😄  I think your friends will like it even more 🚀